tests/unit/UserDefinedFormControllerExtensionTest.php

```diff
@@ -2,21 +2,199 @@
 
 namespace Firesphere\PartialUserforms\Tests;
 
-use Firesphere\PartialUserforms\Extensions\UserDefinedFormControllerExtension;
-use SilverStripe\Core\Injector\Injector;
-use SilverStripe\Dev\SapphireTest;
-use SilverStripe\View\Requirements;
+use Firesphere\PartialUserforms\Models\PartialFormSubmission;
+use SilverStripe\Dev\FunctionalTest;
+use SilverStripe\ORM\FieldType\DBDatetime;
+use SilverStripe\UserForms\Model\UserDefinedForm;
 
-class UserDefinedFormControllerExtensionTest extends SapphireTest
+/**
+ * Class UserDefinedFormControllerExtensionTest
+ * @package Firesphere\PartialUserforms\Tests
+ */
+class UserDefinedFormControllerExtensionTest extends FunctionalTest
 {
-    public function testInit()
+    /**
+     * @var string
+     */
+    protected static $fixture_file = '../fixtures/partialformtest.yml';
+
+    /**
+     * Test that page renders normal UserForm if partial submission is disabled
+     */
+    public function testPartialDisabled()
+    {
+        $form = $this->objFromFixture(UserDefinedForm::class, 'form1');
+        $form->EnablePartialSubmissions = 0;
+        $form->write();
+        $form->publishRecursive();
+
+        $response = $this->get('form-1');
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertContains('form-1/Form', $response->getBody());
+    }
+
+    /**
+     * Test that page renders start form if partial submission is enabled
+     */
+    public function testPartialEnabled()
+    {
+        $form = $this->objFromFixture(UserDefinedForm::class, 'form1');
+        $form->EnablePartialSubmissions = 1;
+        $form->write();
+        $form->publishRecursive();
+
+        $response = $this->get('form-1');
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertContains('Start', $response->getBody());
```

Comment:
`Start` might be a bit generic, we should change this to `form-1/StartForm` to match the test below